package.json

```diff
@@ -20,6 +20,7 @@
     "doc-fix": "run-s markdown-lint-fix prettier-fix",
     "doc-fix-everything": "run-s doc-fix doc-fence-check lint-documentation",
     "doc-fence-check": "tsx tools/check-fenced-code.ts",
+    "doc-validate-helpers": "tsx tools/docs/validate-helpers.ts",
     "lint-documentation": "vitest run --coverage false test/docs/**.spec.ts",
```

Comment:
it should be run here, see other comment below.

we should convert all other doc lint to those spec files too.